### ISSUE TITLE
picard BedToIntervalList  UnsupportedOperationException

### ISSUE THREAD

**Issue by MeHelmy** [broadinstitute/picard#1746]
Installed picard using conda:

`conda install -c bioconda picard`

When I run:
`picard BedToIntervalList -h`

I get this error:

```
WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
Exception in thread "main" java.lang.ExceptionInInitializerError
	at org.broadinstitute.barclay.argparser.CommandLineArgumentParser.handleArgumentAnnotation(CommandLineArgumentParser.java:705)
	at org.broadinstitute.barclay.argparser.CommandLineArgumentParser.createArgumentDefinitions(CommandLineArgumentParser.java:333)
	at org.broadinstitute.barclay.argparser.CommandLineArgumentParser.<init>(CommandLineArgumentParser.java:121)
	at picard.cmdline.CommandLineProgram.getCommandLineParser(CommandLineProgram.java:414)
	at picard.cmdline.CommandLineProgram.getCommandLineParserForArgs(CommandLineProgram.java:400)
	at picard.cmdline.CommandLineProgram.parseArgs(CommandLineProgram.java:348)
	at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:223)
	at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:103)
	at picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:113)
Caused by: java.lang.UnsupportedOperationException: No class provided, and an appropriate one cannot be found.
	at org.apache.logging.log4j.LogManager.callerClass(LogManager.java:576)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:601)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:588)
	at org.broadinstitute.barclay.argparser.NamedArgumentDefinition.<clinit>(NamedArgumentDefinition.java:17)
	... 9 more
```

Thank you!

**Comment by MykMal** [comment]
This seems to be the same bug as #1747.

**Comment by MeHelmy** [comment]
@MykMal  Thank you! 
So the solution is to use an older version of Picard?

Best,
Medhat

**Comment by MykMal** [comment]
@MeHelmy Yes, from what I can tell this error is specific to Picard 2.26.5. For now you can try using the previous version instead.

**Comment by gbggrant** [comment]
A fix for this has been released in Picard 2.26.6.  Apologies for the error.
